<?xml version="1.0"?>

<robot name="lidar_camera">

    <link name="base_link"/>
    
    <joint name="base_to_camera" type="fixed">
        <parent link="base_link"/>
        <child link="camera_link"/>
        <origin xyz="0.0 -0.135 -0.145" rpy="1.57079 0.0 1.57079"/>
    </joint>

    <link name="camera_link"/>

    <joint name="camera_to_optical" type="fixed">
        <parent link="camera_link"/>
        <child link="camera_rgb_optical_frame"/>
        <origin xyz="0.0 0 0.0" rpy="0 0 0" />
    </joint>

    <link name="camera_rgb_optical_frame" />

    <joint name="base_to_lidar" type="fixed">
        <parent link="base_link"/>
        <child link="lidar"/>
        <origin xyz="0.0 0.0 -.003" rpy="0.0 0.0 0.0"/>
    </joint>

    <link name="lidar"/>

</robot>
